src/server/server.ts

```diff
@@ -185,6 +190,27 @@ async function serveArticlePost(
 	}
 }
 
+async function serveEditionArticlePost(
+	req: Request,
+	res: ExpressResponse,
+	next: NextFunction,
+): Promise<void> {
+	try {
+		// req.body: should contain a direct response from capi
+		// fetched by the edition backend
+		const searchResponse = await capiSearchDecoder(req.body);
+		// this end point serve single article so take the first item only
+		const content = searchResponse.results[0];
```

Comment:
Longer term how do we want to handle this? Are we always going to send a full search response and pull out the first article in the list? Is there an overhead cost associated with that?

Reply:
I have refactored this and now it receives a single 'Content' object from the payload